dissect/target/plugins/os/windows/datetime.py

```diff
@@ -191,17 +195,31 @@ def tzname(self, dt: datetime) -> str:
 class DateTimePlugin(Plugin):
     __namespace__ = "datetime"
 
+    TZ_INFO_KEY = "HKLM\\SYSTEM\\CurrentControlSet\\Control\\TimeZoneInformation"
+    TZ_LIST_KEY = "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Time Zones"
+    TZ_NAME_ENGLISH_KEY = "TimeZoneKeyName"  # >= Windows 7
+    TZ_NAME_LOCALIZED_KEY = "StandardName"  # < Windows 7 (https://github.com/dateutil/dateutil/issues/210)
+    TZ_NAME_LOCALIZED_KEY_MATCH = "Std"
+
     def __init__(self, target):
         super().__init__(target)
 
-        tz_info_key = "HKLM\\SYSTEM\\CurrentControlSet\\Control\\TimeZoneInformation"
+        tz_name = None
 
+        timezone_information = self.target.registry.key(self.TZ_INFO_KEY)
         try:
-            tz_name = self.target.registry.key(tz_info_key).value("TimeZoneKeyName").value
-            self._tzinfo = self.tz(tz_name)
-        except RegistryError:
+            tz_name = timezone_information.value(self.TZ_NAME_ENGLISH_KEY).value  # Windows 7+
+        except RegistryValueNotFoundError:
+            tz_name_localized = timezone_information.value(self.TZ_NAME_LOCALIZED_KEY).value
+            for timezone_key in self.target.registry.key(self.TZ_LIST_KEY).subkeys():
+                if tz_name_localized == timezone_key.value(self.TZ_NAME_LOCALIZED_KEY_MATCH).value:
```

Comment:
I personally find it harder to follow with the value names as class constants. The key paths I get, but perhaps keep the value names in the code? They're also used only once.

Reply:
Ok, will fix.